bundle/manifests/node-maintenance-operator.clusterserviceversion.yaml

```diff
@@ -240,5 +240,6 @@ spec:
       - nodemaintenances
     sideEffects: None
     targetPort: 9443
+    timeoutSeconds: 20
```

Comment:
I think this was 15 on master?

Reply:
good catch. I was experimenting with this value while trying to get tests green